verification/epic.go

```diff
@@ -135,7 +139,6 @@ func (e *EpicClassifier) Verify(params *Params) (*Results, error) {
 		Source:         sourcePath,
 		Renditions:     renditions,
 		OrchestratorID: oid,
-		Model:          "https://storage.googleapis.com/verification-models/verification.tar.xz",
```

Comment:
Shouldn't we remove `Model` field from `epicRequest` entirely?

Reply:
Fixed in 41fbc8c